pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -809,6 +815,146 @@ func GatherMachineSet(i *Gatherer) func() ([]record.Record, []error) {
 	}
 }
 
+// GatherInstallPlans collects Top x InstallPlans.
+// Because InstallPlans have unique generated names, it groups them by namespace and the "template"
+// for name generation from field generateName.
+// It also collects Total number of all installplans and all non-unique installplans.
+//
+// The Kubernetes api https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/certificates/v1beta1/certificatesigningrequest.go#L78
+// Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#certificatesigningrequestlist-v1beta1certificates
+//
+// Location in archive: config/certificatesigningrequests/
+func GatherInstallPlans(i *Gatherer) func() ([]record.Record, []error) {
+	return func() ([]record.Record, []error) {
+		var plansBatchLimit int64 = 500
+		cont := ""
+		recs := map[string]collectedPlan{}
```

Comment:
This should be probably `recs := map[string]*collectedPlan{}`. I tried to update the test by adding one same installplan (only with different name) and the corresponding count was only 1 in the gathered map (should be 2 in this case AFAIK).